vscode/microsoft-kiota/l10n/bundle.l10n.ar.json

```diff
@@ -77,5 +77,10 @@
   "Migration completed, but no changes were detected.": "تم الانتهاء من الترحيل، ولكن لم يتم اكتشاف أي تغييرات.",
   "Migration failed":"فشل الترحيل",
   "API clients migrated successfully!":"تم ترحيل عملاء API بنجاح!",
-  "Could not determine the workspace folder.": "لا يمكن تحديد مجلد مساحة العمل."
+  "Could not determine the workspace folder.": "لا يمكن تحديد مجلد مساحة العمل.",
+  "Client": "",
```

Comment:
Have you tested this strings? Am afraid leaving the translations as empty strings means message displayed would be empty. Rather than have no key for it so that the english translation would still be used.

Reply:
This depends on the speed of response from the translation team. If we need the changes to go in faster, they can go without the translations, then the translations can come in later

Reply:
I meant remove the keys if they don't have a translation yet. Otherwise you are instructing the translations to be empty strings which is not the case.

Reply:
This was understood. Depending on the number of strings received by a logical time to close the task, the rest will be removed